src/FINDOLOGIC/Export/CSV/CSVItem.php

```diff
@@ -92,8 +106,15 @@ private function buildAttributes()
     private function buildImages()
     {
         // Use the first available image that is not restricted by usergroup.
```

Comment:
Comment says first available image is used. But if there are two or more images, the exception is thrown. Either update comment or (I think this one is better) just use the first image in the array.